cpp/SnowflakeAzureClient.cpp

```diff
@@ -49,14 +49,16 @@ SnowflakeAzureClient::SnowflakeAzureClient(StageInfo *stageInfo,
       snowflake_global_get_attribute(SF_GLOBAL_CA_BUNDLE_FILE, caBundleFile, sizeof(caBundleFile));
       CXX_LOG_TRACE("ca bundle file from SF_GLOBAL_CA_BUNDLE_FILE *%s*", caBundleFile);
   }
-if( caBundleFile[0] == 0 ) {
+  else if( caBundleFile[0] == 0 ) {
```

Comment:
Why the else if, don't you always want to check this.

Reply:
Thats true, The else in line 48 also needs to be removed.